src/main/scala/com/fulcrumgenomics/alignment/Alignment.scala

```diff
@@ -82,4 +120,55 @@ case class Alignment(query: Array[Byte],
 
   /** One based closed coordinate of the end of the alignment on the query sequence. */
   def targetEnd: Int = targetStart + cigar.lengthOnTarget - 1
+
+  /**
+    * Generates a padded text representation of the alignment for visualization. The returned
+    * sequence will consist of three lines as follows (minus the labels on the left):
+    *
+    * query : ACGTGAACTGACT-ACTGTATGCG
+    * align : |||||  |||||| ||||||||.|
+    * target: ACGTG--CTGACTGACTGTATGGG
```

Comment:
I'm happy to parameterize it so it's possible to use whatever character you like.  I'm pretty used to seeing `-` though in multiple alignments.